types/event_bus.go

```diff
@@ -185,11 +209,11 @@ func (NopEventBus) UnsubscribeAll(ctx context.Context, subscriber string) error
 	return nil
 }
 
-func (NopEventBus) PublishEventNewBlock(data EventDataNewBlock) error {
+func (NopEventBus) PublishEventNewBlock(data EventDataNewBlock, tags []cmn.KVPair) error {
```

Comment:
This also depends on the question of whether to include begin/end results in the event data. Because in case we can include them, this extra argument is not required.

Reply:
Exactly. Let's me chew on this idea for a day

Reply:
Any updates?